### PR TITLE
Make better & more query types

### PR DESCRIPTION
**Pull request by lesics**

### CODE REVIEW

File: wikidataQuery.py [hunk 1]
```diff
@@ -165,22 +171,38 @@ def submitTypeQuery(entity_id, properties_id, query_type):
             print("Problem with the following query:")
             print(query)
             print(traceback.format_exc())
-        return []
+        return []
```
Comment (nvbln) [inline wikidataQuery.py]: Gotta love those random tabs. ;)

File: wikidataQuery.py [hunk 2]
```diff
@@ -3,13 +3,17 @@
 import settings
 import traceback
 import json
+import simplejson
```
Comment (nvbln) [inline wikidataQuery.py:6]: Please make clear that this also needs to be installed. Also, could you add both this and TextBlob to the README?

File: wikidataQuery.py [hunk 3]
```diff
@@ -3,13 +3,17 @@
...
+# Installable with 'pip install textblob'
+from textblob import TextBlob
```
Comment (nvbln) [inline wikidataQuery.py:7]: Apparently I also needed to run `python3 -m textblob.download_corpora` please include that in the README as well.

File: wikidataQuery.py [hunk 4]
```diff
@@ -3,13 +3,17 @@
 import settings
 import traceback
 import json
+import simplejson
+# Installable with 'pip install textblob'
```
Comment (nvbln) [inline wikidataQuery.py]: Please refer to `pip3` instead of `pip`. Afaik `pip` will install the 2.7 python version, which creates unwanted confusion.

**Review by nvbln — CHANGES_REQUESTED** [review]

File: wikidataQuery.py (hunk 2 above)
Comment (lesics) [reply wikidataQuery.py:6]: simplejson does not need to be installed though

File: wikidataQuery.py (hunk 3 above)
Comment (lesics) [reply wikidataQuery.py:7]: Hm, weird. I didn't need 'python3 -m textblob.download_corpora'

File: wikidataQuery.py (hunk 4 above)
Comment (lesics) [reply wikidataQuery.py]: I referred to pip, because I don't have pip3 command, so I can't tell how it'll react. pip worked for me just fine

**Review by tomveldhuis — COMMENTED** [review]
I have some changes to the README on the todos2 branch as well, so maybe I should push that first?

**Comment by nvbln** [comment]
> I have some changes to the README on the todos2 branch as well, so maybe I should push that first?

I suppose you're too late since it's already been committed. If you just look out for merge conflicts then it should be fine.

**Review by nvbln — APPROVED** [review]

**Pull request merged**